mypy/suggestions.py

```diff
@@ -580,6 +580,24 @@ def visit_tuple_type(self, t: TupleType) -> str:
         s = self.list_str(t.items)
         return 'Tuple[{}]'.format(s)
 
+    def visit_callable_type(self, t: CallableType) -> str:
+        args = []
+        for kind, name, typ in zip(t.arg_kinds, t.arg_names, t.arg_types):
+            arg = typ.accept(self)
+            if kind == ARG_STAR:
+                arg = '*' + arg
```

Comment:
This doesn't actually work? `Callable[[*int], int]` is an error. You could "extended callable types" instead.